app/posts/personalised-prevention-platform/2025/04/2025-04-11-onboarding-users.md

```diff
@@ -42,9 +42,11 @@ We also made the following assumptions:
 - our scenario starts from an NHS.UK calculator
 - users are already somehow digitally registered ‘with the NHS’
 
-<figure class="govuk-!-margin-top-7">
-  <img src="prototype-flow.png" alt="Diagram depicting the prototype user journey using sequential screen grabs from left to right." style="width: 92vw; max-width: 960px;">
-  <figcaption>The prototype journey from left to right, from calculator result to a ‘finding services’ loading screen.</figcaption>
+<figure class="nhsuk-image" style="width: 92vw; max-width: 960px; ">
+  <img class="nhsuk-image__img" src="prototype-flow.png" alt="Diagram depicting the prototype user journey using sequential screen grabs from left to right.">
+  <figcaption class="nhsuk-image__caption">
+    The prototype journey from left to right, from calculator result to a ‘finding services’ loading screen.
+  </figcaption>
 </figure>
```

Comment:
Switch to the markdown for [Figure with caption](https://design-history.prevention-services.nhs.uk/guide/using-markdown/#images-with-captions)?

Would be something like this:

```
![Diagram depicting the prototype user journey using sequential screen grabs from left to right.](prototype-flow.png "The prototype journey from left to right, from calculator result to a ‘finding services’ loading screen.")
```

Reply:
This bit `style="width: 92vw; max-width: 960px; "` is kind of important?

How might we enable a "full width" feature?